tests/vm_test.go

```diff
@@ -2163,3 +2163,18 @@ func haveStateChangeRequests() gomegatypes.GomegaMatcher {
 		}),
 	}))
 }
+
+func newRandomVirtualMachineInstanceWithFileDisk(imageUrl, namespace string, accessMode k8sv1.PersistentVolumeAccessMode) (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
+	if !libstorage.HasCDI() {
+		Skip("Skip DataVolume tests when CDI is not present")
+	}
+	sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+	if accessMode == k8sv1.ReadWriteMany {
+		sc, foundSC = libstorage.GetRWXFileSystemStorageClass()
+	}
+	if !foundSC {
+		Skip("Skip test when Filesystem storage is not present")
+	}
+
+	return tests.NewRandomVirtualMachineInstanceWithDisk(imageUrl, namespace, sc, accessMode, k8sv1.PersistentVolumeFilesystem)
```

Comment:
Let's also use libvmi, if we're changing this function.

Reply:
updated